backend/app/anti_recommendation_engine/anti_recommendation_engine.py

```diff
@@ -39,42 +37,45 @@ def __select_anti_recommender(self) -> AntiRecommender | None:
         """Select and return an AntiRecommender based on values stored in settings."""
 
         if (
-            settings.anti_recommender_type.lower() == "openai"
+            settings.anti_recommender_type == AntiRecommenderType.OPEN_AI
             and settings.openai_api_key
         ):
-            return NormalOpenAiAntiRecommender()
+            return NormalOpenaiAntiRecommender()
+
+        if settings.anti_recommender_type == AntiRecommenderType.ARKG:
+            return ArkgAntiRecommender(
+                arkg_base_iri=settings.arkg_base_iri,
+                arkg_file_path=settings.arkg_file_path,
+                arkg_mime_type=settings.arkg_mime_type,
+                record_keys=tuple(self.__records_by_key.keys()),
+            )
 
         return None
 
-    def get_initial_records(self, record_type: RecordType) -> tuple[Record, ...]:
+    def get_initial_records(self) -> tuple[Record, ...]:
```

Comment:
Nit: in Python you usually don't prefix with `get_`.

Reply:
I have removed this prefix from the codebase.